ui/src/pages/Input/InputPage.tsx

```diff
@@ -158,16 +215,14 @@ function InputPage() {
             serviceName: row.serviceName,
             stanzaName: row.name,
             formLabel: mode === MODE_CLONE ? `Clone ${label}` : `Update ${label}`,
-            mode,
+            mode: mode as Mode,
```

Comment:
mode already has a type of Mode, there is no sense to declare it

Reply:
removed